From d2498d4020cef200123131f7fe27c881fa747c17 Mon Sep 17 00:00:00 2001
From: caodongxia <315816521@qq.com>
Date: Wed, 26 May 2021 11:21:15 +0800
Subject: [PATCH] create patch

---
 testsuite/modules.50-cmds/375-deps6.exp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/testsuite/modules.50-cmds/375-deps6.exp b/testsuite/modules.50-cmds/375-deps6.exp
index 0add1bb..c94fe24 100644
--- a/testsuite/modules.50-cmds/375-deps6.exp
+++ b/testsuite/modules.50-cmds/375-deps6.exp
@@ -836,7 +836,7 @@ lappend ans [list setpath MODULES_LMPREREQ mtf&$mp/msf]
 lappend ans [list setpath MODULES_LMCONFLICT $mp/msf&$mp/mtf]
 lappend ans [list setpath MODULES_LMNOTUASKED $mp/msf]
 #FIXME: top load messages for mtf should contain a warning to the forced conflict with $mp/msf
-testouterr_cmd_re sh {load --force mtf} $ans [msg_load $mpre/msf [err_conloif $mpre/mtf]]\n\n[msg_load mtf "Loading requirement: $mpre/msf"]
+testouterr_cmd_re sh {load --force mtf} $ans [msg_load $mp/msf [err_conloif $mp/mtf]]\n\n[msg_load mtf "Loading requirement: $mp/msf"]
 
 
 #
-- 
2.27.0

